cucumber-jparest/src/main/java/uk/gov/homeoffice/digital/sas/cucumberjparest/api/JpaRestApiClient.java

```diff
@@ -78,32 +75,25 @@ private static URI getUri(@NonNull RequestSpecification requestSpecification) {
     return requestUri;
   }
 
-  @Getter
-  private final ServiceRegistry serviceRegistry;
-
-  @Autowired
-  public JpaRestApiClient(ServiceRegistry serviceRegistry) {
-    this.serviceRegistry = Objects.requireNonNull(serviceRegistry,
-        "serviceRegistry must not be null");
-  }
-
   /**
    * Creates resources in the specified service using the provided payload.
    *
    * @param persona  The persona making the request.
+   * @param tenantId The tenant ID; if not supplied fallback on system property
    * @param service  The name of the service where the resources will be created. The service name
    *                 must exist in the {@link ServiceRegistry}
    * @param resource The name of the type of resource to create
    * @param payload  The resource to create
    * @return JpaRestApiResourceResponse
    */
-  public JpaRestApiResourceResponse create(Persona persona, String service, String resource,
-      String payload) {
+  public JpaRestApiResourceResponse create(Persona persona, String tenantId, String service,
+      String resource, String payload) {
     URI uri = getResourceUri(service, resource);
     RequestSpecification spec = given()
         .baseUri(uri.toString())
         .body(payload)
-        .queryParam(TENANT_ID_PARAM_NAME, tenantId);
+        .queryParam(TENANT_ID_PARAM_NAME,
```

Comment:
I was going to say that we should just annotate the param to not allow nulls but then I see by default it looks like we've added the param and always pass null in all of the calling methods.
It doesn't feel like a good pattern.